src/components/ContractDetailsPanel.vue

```diff
@@ -133,6 +157,8 @@ import CopyChip from '@/components/CopyChip'
 import AppChip from '@/components/AppChip'
 import { formatAePrice, formatAettosToAe, formatEllipseHash } from '@/utils/format'
 import DatetimeLabel from '@/components/DatetimeLabel'
+import { contractsHints } from '~/utils/hints/contractsHints'
```

Comment:
I think we are using the wrong alias here and in other places

Reply:
fixed